conf/nginx/nginx.conf

```diff
@@ -60,6 +67,26 @@ http {
             add_header "X-Via" "static-proxy, redirect";
         }
 
+        location @proxy_not_found {
+            # Not found / gone => 404 not found
+            try_files /proxy/not_found.html =404;
```

Comment:
These files don't have to exist yet. NGINX will try them, not find them and fallback to internal defaults.

Reply:
Do we want to proxy 404s? I guess why not--these won't be considered as "errors" by monitoring services like Elastic Beanstalk's, they shouldn't cause any false alarms in monitoring or alerting

Reply:
I think 404 is one of the few HTTP codes which are understood by lay-people, so it's probably a good idea.